File: C11/백준/Bronze/19602. Dog Treats/Dog Treats.c
#include <stdio.h>

int main(void) {
	
	int S, M, L;
	scanf("%d %d %d", &S, &M, &L);
	
	int happiness = S + 2 * M + 3 * L;
	if (happiness >= 10) {
		printf("happy");
	} else {
		printf("sad");
	}
	
	return 0;
}

File: Java/백준/Bronze/19602. Dog Treats/Dog Treats.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int S = Integer.parseInt(br.readLine());
        int M = Integer.parseInt(br.readLine());
        int L = Integer.parseInt(br.readLine());

        int happiness = S + 2 * M + 3 * L;
        if (happiness >= 10) {
            System.out.print("happy");
        } else {
            System.out.print("sad");
        }
    }
}